setup.py

```diff
@@ -0,0 +1,40 @@
+"""A setuptools based setup module.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+import os
+
+from setuptools import setup, find_packages, Extension
+
+# pylint: disable=redefined-builtin
+
+here = os.path.abspath(os.path.dirname(__file__))  # pylint: disable=invalid-name
+
+NUMENC_MODULE = Extension('numenc_module', sources=['numenc/encoder_decoder.c'])
+
+with open(os.path.join(here, 'README.rst'), encoding='utf-8') as f:
+    long_description = f.read()  # pylint: disable=invalid-name
+
+setup(
+    name='pynumenc',
+    version='1.0.0',
```

Comment:
Please make numenc_meta.py module to specify the version, description *etc*. See for example, how pypackagery handles the versions.